src/Layout/Header/index.jsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react'
-import styles from '../../styles.module.css'
+import styles from './styles.module.css'
 import { TopNavigationBar } from './Top Navigation'
 import { SearchBox } from './SearchBox'
 
-export function Header () {
+export function Header ({ children }) {
   return (
     <div className={styles.header}>
       <TopNavigationBar />
       <SearchBox />
+      { children }
```

Comment:
I don't think we need this. You are not passing any `children` to `Header` component anyway.